app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+  <%= form_with(model: expense) do |form| %>
+    <div>
+      <h5 class='text-center display-6'>Add new transaction</h5>
+    </div>
+    <%= render "devise/shared/error_messages", resource: expense %>
+    <div class="container">
+      <%= form.text_field :name, autofocus: true, placeholder: "Name", class: "form-control m-2", required: true %>
+      <%= form.number_field :amount, placeholder: "Amount", class: "form-control  m-2", required: true %>
```

Comment:
- [ ] **[OPTIONAL]** One is unable to add transactions with decimal places, like `3.4 or 0.99` because of the generic number type input accepts integers. It would be nice to allow for decimal places, by adding the `step: 'any'` to your 
number input, like so:
  
  ```
  <%= form.number_field :amount, placeholder: "Amount", class: "form-control  m-2", required: true, step: 'any' %>
  ```

Reply:
Implemented